src/filesystem/Parser.cpp

```diff
@@ -107,6 +107,8 @@ static struct options {
   bool allowOther;
 
   int qsfsMultiThread;       // internal use only to turn on multi thread
+
+  mode_t fallbackMode;     // default fallback mode
```

Comment:
And could you also update the docs in HelpText.cpp::ShowQSFSHelp and HelpText.cpp::ShowQSFSUsage?

Reply:
I think this option is relating to the stat, we better group this new option with other stat releated options (maxstat and statexpire). I mean we better to add this new option following statexpire, this also applies to your changes in other files.